spec/factories.rb

```diff
@@ -1,6 +1,8 @@
 FactoryGirl.define do
+
   factory :photo do
-    img_src "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/00829/opgs/edr/fcam/FRB_471079934EDR_F0442062FHAZ00323M_.JPG"
+    img_src "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/"+
+            "surface/sol/00829/opgs/edr/fcam/FRB_471079934EDR_F0442062FHAZ00323M_.JPG"
```

Comment:
Line is too long. [86/80]<br>Use 2 (not 8) spaces for indenting an expression spanning multiple lines.